chapters/arrays.tex

```diff
@@ -532,8 +532,8 @@ \subsection{Matrix and Vector Algebra Functions}\doublelabel{matrix-and-vector-a
 & Returns the outer product of vectors v1 and v2 ( = matrix(v1)*transpose(
 matrix(v2) ) ).\\ \hline
 \lstinline!symmetric(A)!
-& Returns a matrix where the diagonal elements and the elements above the
-diagonal are identical to the corresponding elements of matrix A and
+& Returns a square matrix where the diagonal elements and the elements above the
+diagonal are identical to the corresponding elements of the square matrix A and
 where the elements below the diagonal are set equal to the elements
 above the diagonal of A, i.e., \lstinline!B := symmetric(A) ->!
```

Comment:
As said in the issue, this textual description of `symmetric` is a bit confusing, perhaps partly because it is almost too verbose to be of any help.  What about:
> Constructs a symmetric matrix which is identical to the square matrix `A` on and above the diagonal, i.e., …

Reply:
That seems like a good improvement, but I kept "Returns".

Reply:
Did you forget to push the change?  At least it doesn't seem to be in the same commit as the one with the Real/Integer clarification.

Reply:
I don't see this being resolved by the commit _Main clarifications._, so I'm still wondering…

Reply:
Must have been missed, added now.

Reply:
Good.  Resolving.